src/modules/portfolio/.fuse_hidden0000009000000001

```diff
@@ -0,0 +1,90 @@
+.portfolio {
```

Comment:
a strange file

Reply:
name I mean

Reply:
I don't know what it is. Probably css files doubled. What I need to do? Just delete on my computer?

Reply:
be careful. Yes jsut delete that

Reply:
Just delete like usual files? Or i need to use git rm?